### ISSUE TITLE
Rule no-extraneous-dependencies complains about non-existent tippy-react-headless

### ISSUE THREAD

**Issue by klimashkin** [benmosher/eslint-plugin-import#2106]
After upgrading the plugin from 2.22.0 to 2.23.3, the rule `no-extraneous-dependencies` complains that
```
 ✘  https://google.com/#q=import%2Fno-extraneous-dependencies  
‘tippy-react-headless’ should be listed in the project’s dependencies. Run ‘npm i -S tippy-react-headless’ to add it
```
when we import `@tippyjs/react/headles` like that according to the [documentation](https://github.com/atomiks/tippyjs-react#headless-tippy)
```jsx
import React from 'react';
import Tippy from '@tippyjs/react/headless'; // different import path!

const HeadlessTippy = () => (
  <Tippy
    render={attrs => (
      <div className="box" tabIndex="-1" {...attrs}>
        My tippy box
      </div>
    )}
  >
    <button>My button</button>
  </Tippy>
);
```
But package tippy-react-headless doesn't exist in npm at all

**Comment by tarlepp** [comment]
related - https://github.com/benmosher/eslint-plugin-import/pull/2097

**Comment by klimashkin** [comment]
Fixed in 2.23.4
